server.js

```diff
@@ -0,0 +1,192 @@
+const express = require("express");
+const cors = require("cors");
+
+const db = require("./data/db");
+
+const port = 5555;
+const server = express();
+
+server.use(express.json());
+server.use(cors());
```

Comment:
Remember that if you don't specify an origin url in `server.use(cors())` it will default the origin to Open Access. 
You want something like this ```server.use(cors({ origin: 'http://localhost:3000' }));```.

Reply:
understood, noted!